src/audio/dsm/sof_dsm.c

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Is this file meant to be generic DSM or Maxim specific DSM ? If its the latter we should rename to maxim_dsm.c or similar.

Reply:
Thanks. Shall fix the name.